src/compiler/parser.ts

```diff
@@ -5970,6 +5970,9 @@ namespace ts {
                 typeArguments = (expression as ExpressionWithTypeArguments).typeArguments;
                 expression = (expression as ExpressionWithTypeArguments).expression;
             }
+            if (token() === SyntaxKind.QuestionDotToken) {
+                parseErrorAtCurrentToken(Diagnostics.Invalid_optional_chain_from_new_expression);

```

Comment:
A question I have is whether we should emit the error for all targets. For <= ES2019, the downleveling we do means there won't be a syntax error at runtime. @rbuckton